joomla/joomla-cms
# Implement ACL to backend menus

#### Summary of Changes

With this pull request it's possible to define the ACL rights for each menutype (and therefore all menu items of this type).
#### Testing Instructions
- Apply patch
- Repair the database (Extensions => Manage => Database)
- Go to "Menus" => "Manage"
- Open a menutype entry (not a menu item entry) e.g. "mainmenu"
- A new tab  "Menu Permissions" appears
- Define your ACL rights
- Test with menu items
